api/.env.dev

```diff
@@ -6,11 +6,11 @@ POSTGRES_DB="hub"
 POSTGRES_USER="postgres"
 POSTGRES_PASSWORD="postgres"
 
-GH_CLIENT_ID=""
-GH_CLIENT_SECRET=""
+GH_CLIENT_ID="client-id"
```

Comment:
Do we need this change ??

Reply:
every time we need to run the server we have to add some values let some default values be there. its in dev so won't affect anything, we can add in development.md to change as per requirement